src/app/space/app-launcher/services/app-launcher-project-summary.service.ts

```diff
@@ -27,4 +78,39 @@ export class AppLauncherProjectSummaryService implements ProjectSummaryService {
   verify(summary: Summary): Observable<boolean> {
     return Observable.of(true);
   }
+
+  private handleError(error: Response | any) {
+    // In a real world app, we might use a remote logging infrastructure
```

Comment:
Please remove this line - it's not true and leads to other developers copying a common function that doesn't make this project look good.